test/inductor/test_config.py

```diff
@@ -32,15 +32,15 @@ def tearDown(self):
     def test_set(self):
         config.max_fusion_size = 13337
         self.assertEqual(config.max_fusion_size, 13337)
-        self.assertEqual(config.to_dict()["max_fusion_size"], 13337)
-        config.to_dict()["max_fusion_size"] = 32
+        self.assertEqual(config.shallow_copy_dict()["max_fusion_size"], 13337)
```

Comment:
Hmm, is this BC breaking?

Reply:
I added a deprecation warning to `to_dict`